app/helpers/payment_details_helper.rb

```diff
@@ -61,8 +61,8 @@ def stripe_billing_address
       city: billing_address.city,
       state: billing_address.state,
       postal_code: billing_address.zip,
-      country: billing_address.country
-    }.transform_values { |value| value.presence || '' }
+      country: country_code_for(billing_address.country)
+    }.delete_if { |key, value| value.blank? }
```

Comment:
Noice!

Reply:
Hm, but `.compact` is not equivalent. It would be if it was `delete_if { |_, value| value.nil? }` rather than `.blank?`. The question is - do we want to delete blank values also?

Reply:
True 😓 I guess those will never be empty though, because according to validation they can't be. I guess the most important part is that they are not sent `nil`. I will double check this.